client/src/webview-js/cnxml-preview/cnxml-preview.js

```diff
@@ -177,35 +359,23 @@ function __vdom__applyProps($el, newProps, oldProps = {}) {
     const newValue = newProps[name]
     const oldValue = oldProps[name]
     if (__vdom__isObject(newValue)) {
+      /* istanbul ignore next */
       throw new Error('Does not support setting multiple attributes on an element')
       // applyProps($el[name], (newValue as any) as ObjectLiteral, (oldValue as any) as ObjectLiteral)
     } else {
       if (!newValue) {
-        __vdom__removeProp($el, name)
+        $el.removeAttribute(name)
       } else if (newValue !== oldValue) {
-        __vdom__setProp($el, name, newValue)
+        $el.setAttribute(name, newValue)
       }
     }
   })
 }
-function __vdom__setProp($el, name, value) {
```

Comment:
Yeah, our code doesn't add/remove classes and if it did we could just write `'class': ...`